### PR TITLE
Create Add wstETH-C as new Vault type - April 25, 2022.md

### PR DESCRIPTION
**Pull request by Gala-G**

### CODE REVIEW

File: governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Add wstETH-C as a new Vault Type - April 25, 2022
+summary: This poll proposes system parameters which could be used to initialize wstETH-C as a new vault type.
+discussion_link: https://forum.makerdao.com/t/introduction-risk-parameters-for-wsteth-c-vault/14669
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
```
Comment (patrick-j-govalpha) [inline governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md:8]: Can you add the "High Impact" category as well, please?

File: governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding wstETH-C (Lido Staked ETH) as a new vault type in the Maker Protocol with the parameters below:
+
+### Initial Collateral Parameters
+
+* Underlying Collateral: wstETH
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 0.75%
+* [Liquidation Ratio](https://community-development.makerdao.com/en/learn/governance/param-liquidation-ratio): 185%
+* [Debt Ceiling (`line`)](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 150 million DAI
```
Comment (patrick-j-govalpha) [inline governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md:34]: For DC-IAM vaults, which this is, this should be "Maximum Debt Ceiling (`line`)"

File: governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam): 15 million DAI
+* [Ceiling Increase Cooldown (`ttl`)](https://makerdao.world/en/learn/governance/module-dciam): 8h
```
Comment (patrick-j-govalpha) [inline governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md:36]: can you change this to "8 hours" please

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Just a couple of small corrections then it should be good to go :)

File: governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md (hunk 1 above)
Comment (Gala-G) [reply governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md:8]: Do we want to change the templates adding -[$Impact Level] ? I can work on that.

**Pull request merged**